.github/workflows/release.yml

```
@@ -9,6 +9,8 @@ jobs:
   release:
     permissions:
       contents: write
+      actions: read
+      actions: write
```

actionlint (1.7.12)

[error] key "actions" is duplicated in "permissions" section. previously defined at line:12,col:7. note that this key is case insensitive

(syntax-check)